lib/process-services-cloud/src/lib/process/process-filters/components/process-filters-cloud.component.html

```diff
@@ -22,12 +22,12 @@
                 </span>
             </div>
             <span
-                *ngIf="counters$[filter.key]"
+                *ngIf="counters[filter.key]"
                 [attr.data-automation-id]="filter.key + '_filter-counter'"
                 class="adf-process-filters__entry-counter"
                 [class.adf-active]="isFilterUpdated(filter.key)"
                >
-                {{ counters$[filter.key] | async }}
+                {{ counters[filter.key] }}
```

Comment:
I don't really like this approach as it makes the whole thing less reactive, as we discussed we can tackle this issue on a different perspective

Reply:
humm point 3 and 4 makes me think I didn't explained the way we might improve this, but as i don't want to drag this around for too long i would say it's ok